pyproject.toml

```diff
@@ -0,0 +1,96 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "tell"
+description = "An open-source Python package to model future hourly total electricity loads."
+readme = "README.md"
+version = "1.2.1"
```

Comment:
Let's bump the version to 1.3.0

This will also need to be added in in the following:

- tell/__init__.py
- tell/install_forcing_data.py (use same data link as previous version, just add new entry)
- tell/install_quickstarter_data.py
- tell/install_raw_data.py
- docs/source/conf.py

Once these are done, we will need to rebuild and deploy the docs, conduct a release, and push the new version to PyPi.  Let's set up actions for these perhaps in a separate PR.

Reply:
updated!